src/ApplicationManager.cpp

```
@@ -500,6 +500,23 @@ void ApplicationManager::handleSpoolDetected(const AppMessage& msg) {
         f.density = s.density;
         f.diameter_mm = s.diameter;
 
+#ifndef NATIVE_TEST
+        BambuTagData bambuData;
+        if (spoolState.kind == TagKind::BambuTag && NFCManager::getInstance().getLastBambuTagData(bambuData)) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)